### PR TITLE
#154 Add SES req in volume 1.

### PR DESCRIPTION
**Pull request by d-gregorczyk**
## 📑 Description

<add description if available>

## ☑ Mandatory Tasks

The following aspects have been respected by the pull request assignee and at least one reviewer:
  
- Changelog update (necessity checked and entry added or not added respectively)
  * [x] Pull Request Assignee
  * [ ] Reviewer

### CODE REVIEW

File: asciidoc/volume1/use-cases/tf1-ch-c-use-case-stad.adoc [hunk 1]
```diff
@@ -80,3 +80,22 @@ image::../images/vol1-diagram-use-case-stad-tech-view.svg[align=center]
...
+====
+NOTE: Clocks of <<vol1_spec_sdpi_p_actor_somds_participant>>s run apart due to lack of synchronization with NTP servers, different clock drifts or cyberattacks.
+
+NOTE: Typically, a <<term_manufacturer>> assumes that system functionality in general may not be available due to loss of network connections, and risks are mitigated accordingly.
+====
+****
+
+Requirement: The manufacturer of a SOMDS Participant SHALL consider the risk of a workflow interruption due to misaligned clocks.
+Note:
+Note:
```
Comment (AnnaFeiler) [inline asciidoc/volume1/use-cases/tf1-ch-c-use-case-stad.adoc:101]: General question: How do we deal with references to the PKPs? In this case, BPKP already contains time sync related RCMs. Do we need to mention that?

Comment (d-gregorczyk) [reply asciidoc/volume1/use-cases/tf1-ch-c-use-case-stad.adoc]: We can add a reference to RCMs of the BPKP. In terms of requirements interoperability/traceability, at some point we have to annotate requirement blocks such that we can formally link to other standards.

The former we can do  today, the latter is something for _tomorrow_.

File: asciidoc/volume1/use-cases/tf1-ch-c-use-case-stad.adoc [hunk 2]
```diff
@@ -80,3 +80,22 @@ image::../images/vol1-diagram-use-case-stad-tech-view.svg[align=center]
...
+.Notes
+[%collapsible]
+====
+NOTE: Clocks of <<vol1_spec_sdpi_p_actor_somds_participant>>s run apart due to lack of synchronization with NTP servers, different clock drifts or cyberattacks.
+
+NOTE: Typically, a <<term_manufacturer>> assumes that system functionality in general may not be available due to loss of network connections, and risks are mitigated accordingly.
+====
+****
+
+Requirement: The manufacturer of a SOMDS Participant SHALL consider the risk of a workflow interruption due to misaligned clocks.
```
[the remark this answers is not in the document]
Comment (d-gregorczyk) [reply asciidoc/volume1/use-cases/tf1-ch-c-use-case-stad.adoc]: Removed

File: asciidoc/volume1/use-cases/tf1-ch-c-use-case-stad.adoc (hunk 1 above)
Comment (AnnaFeiler) [reply asciidoc/volume1/use-cases/tf1-ch-c-use-case-stad.adoc]: Adding a reference to: BPKP RR1162: The MANUFACTURER of an SDC BASE CONSUMER SHALL consider the RISKs resulting from erroneous timestamps. would be great.

**Review by AnnaFeiler — APPROVED** [review]

**Pull request merged**